server/routes/induction/update/additionalTrainingUpdateController.test.ts

```diff
@@ -217,17 +217,17 @@ describe('additionalTrainingUpdateController', () => {
       req.session.inductionDto = inductionDto
 
       const additionalTrainingForm = {
-        additionalTraining: [AdditionalTrainingValue.FULL_UK_DRIVING_LICENCE, AdditionalTrainingValue.OTHER],
-        additionalTrainingOther: 'Beginners cookery for IT professionals',
+        additionalTraining: [AdditionalTrainingValue.HGV_LICENCE, AdditionalTrainingValue.OTHER],
```

Comment:
This test was flawed and slightly misleading before, because it didnt actually update/change anything on the page! The values that were being set in the form were the same values as in the loaded Induction, so whilst it did update the induction, it updated it with the same values.
I've made this change so that we can expect and see a changed Induction when it is sent to the service